properties.schema

```diff
@@ -258,61 +258,71 @@
             "title": "Menu item background colour",
             "type": "string",
             "inputType": "ColourPicker",
-            "default": ""
+            "default": "",
+            "help": "Sets the background colour for the Menu Item."
           },
           "menu-item-inverted": {
             "title": "Menu item foreground colour",
             "type": "string",
             "inputType": "ColourPicker",
-            "default": ""
+            "default": "",
+            "help": "Defines the Menu Item text colour. Should be a colour that provides good contrast against the Menu Item colour."
           },
           "menu-item-border-color": {
             "title": "Menu item border colour",
             "type": "string",
             "inputType": "ColourPicker",
-            "default": ""
+            "default": "",
+            "help": "Defines the border colour that appears around the Menu Item."
           },
           "menu-item-progress": {
             "title": "Menu item progress fill colour",
             "type": "string",
             "inputType": "ColourPicker",
-            "default": ""
+            "default": "",
+            "help": "Defines the Menu Item specific global progress width fill override colour."
           },
           "menu-item-progress-inverted": {
             "title": "Menu item progress background colour",
             "type": "string",
             "inputType": "ColourPicker",
-            "default": ""
+            "default": "",
+            "help": "Defines the Menu Item specific global progress fill background override colour."
           },
           "menu-item-progress-border": {
             "title": "Menu item progress border colour",
             "type": "string",
             "inputType": "ColourPicker",
-            "default": ""
+            "default": "",
+            "help": "Defines the Menu Item specific border override colour that appears around the progress fill bar."
           },
           "menu-item-btn-color": {
             "title": "Menu item button background colour",
             "type": "string",
             "inputType": "ColourPicker",
-            "default": ""
+            "default": "",
+            "help": "Defines the clickable Menu Item Btn element."
           },
           "menu-item-btn-color-inverted": {
             "title": "Menu item button foreground colour",
             "type": "string",
             "inputType": "ColourPicker",
-            "default": ""
+            "default": "",
+            "help": "Defines the Menu Item Btn text colour. Should be a colour that provides good contrast against the Menu Item Btn colour."
           },
           "menu-item-btn-color-hover": {
             "title": "Menu item button background hover colour",
             "type": "string",
             "inputType": "ColourPicker",
-            "default": ""
+            "default": "",
+            "help": "Defines the Menu Item Btn colour when hovered over."
```

Comment:
I'm not sure this help text really adds anything as the title is sufficiently clear by itself.